### PR TITLE
docs: fix usage.md

### PR DESCRIPTION
**Pull request by misitebao**

### CODE REVIEW

File: docs/docs/usage.md [hunk 1]
```diff
@@ -595,7 +595,7 @@ The above syntax is also supported in `deps`.
 
 NOTE: If you want to call a task declared in the root Taskfile from within an
 [included Taskfile](#including-other-taskfiles), add a leading `:` like this:
-`task: :task-name`.
+`task::task-name`.
```
Comment (andreynering) [inline docs/docs/usage.md:598]: This was actually correct before.

Comment (misitebao) [reply docs/docs/usage.md:598]: So this is expected behavior?

Comment (pd93) [reply docs/docs/usage.md:598]: Maybe this example would be clearer with syntax highlighting:

```yaml
tasks:
  my-task:
    cmds:
      - task: :task-in-parent-file
```

Comment (misitebao) [reply docs/docs/usage.md:598]: Ok, I see, thank you.

**Pull request merged**